code/lib/cli/src/js-package-manager/JsPackageManager.ts

```diff
@@ -130,6 +130,45 @@ export abstract class JsPackageManager {
     }
   }
 
+  /**
+   * Remove dependencies from a project using `yarn remove` or `npm uninstall`.
+   *
+   * @param {Object} options contains `skipInstall`, `packageJson` and `installAsDevDependencies` which we use to determine how we install packages.
+   * @param {Array} dependencies contains a list of packages to remove.
+   * @example
+   * removeDependencies(options, [
+   *   `@storybook/react`,
+   *   `@storybook/addon-actions`,
+   * ]);
+   */
+  public removeDependencies(
+    options: {
+      skipInstall?: boolean;
+      packageJson?: PackageJson;
+    },
+    dependencies: string[]
+  ): void {
+    const { skipInstall } = options;
+
+    if (skipInstall) {
+      const { packageJson } = options;
+
+      dependencies.forEach((dep) => {
+        delete packageJson[dep];
```

Comment:
Shouldn't this delete from packageJson.dependencies or devDependencies?

Reply:
Yup! Fixed in #18905